Arch.tex

```diff
@@ -653,15 +662,17 @@ \subsection{ObsLocTAP}
 allows users to run complex queries using the IVOA Table Access Protocol. The document also 
 prescribes how to register and discover ObsLocTAP services. 
 
-\subsection{EPN-TAP}
+\subsection{EPN-TAP - V2 Draft}
```

Comment:
I had included EPN-TAP because it looked like it would go to PR before the interop and I was hedging. Since it is still draft, this section should be removed as well.

Reply:
You mean not talking about EPN-TAP at all?

Reply:
Correct. As above, I don't want to have to draw a line where some drafts are in and others not. So I'm happy to leave PRs in but would prefer to remove mentioned of WDs.